tests/baselines/reference/privateNamesUnique.errors.txt

```diff
@@ -0,0 +1,20 @@
+tests/cases/conformance/classes/members/privateNames/privateNamesUnique.ts(11,7): error TS2322: Type 'B' is not assignable to type 'A'.
+  Property '#foo' is missing in type 'B'. While type 'B' has a private member with the same spelling, its declaration and accessibility are distinct.
+
+
+==== tests/cases/conformance/classes/members/privateNames/privateNamesUnique.ts (1 errors) ====
+    // @target es6
+    
+    class A {
+        #foo: number;
+    }
+    
+    class B {
+        #foo: number;
+    }
+    
+    const b: A = new B();     // Error: Property #foo is missing
+          ~
+!!! error TS2322: Type 'B' is not assignable to type 'A'.
+!!! error TS2322:   Property '#foo' is missing in type 'B'. While type 'B' has a private member with the same spelling, its declaration and accessibility are distinct.

```

Comment:
Too wordy. I would re-use the private error:
"Types have separate declarations of private property '#foo'.

Reply:
@DanielRosenwasser  wrote this one. Thoughts, Daniel?

How about "Types have separate declarations of private-named property '#foo'"?

We'd discussed naming and landed on 'private-named property' because 'private property' is ambiguous (could mean a property with the 'private' modifier).

> Sadly, private-named properties aren't actually properties, but it was the best we could come up with. "Private-named field"?

Reply:
That sounds fine to me. I don't mind having different terms for the two things, I just don't want the messages to be overlong.

Reply:
I provided a few other suggestions too